XCUITests/HomeButtonTests.swift

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class HomeButtonTests: BaseTestCase {
+
+    override func tearDown() {
+        XCUIDevice.shared.orientation = .portrait
+        super.tearDown()
+    }
+
+    func testGoHome() throws {
+        navigator.openURL("https://www.google.com", waitForLoading: true)
+        waitForExistence(app.buttons["TabToolbar.homeButton"], timeout: 5)
+        XCTAssertTrue(app.buttons["TabToolbar.homeButton"].exists)
+        app.buttons["TabToolbar.homeButton"].tap()
+        navigator.nowAt(NewTabScreen)
+        navigator.performAction(Action.CloseURLBarOpen)
+        waitForTabsButton()
+
+        XCTAssertEqual(app.buttons["TabToolbar.homeButton"].label, "Search")
+        navigator.openURL("https://www.twitter.com", waitForLoading: true)
```

Comment:
Same as above about using a testing site. Sometime this site takes to long to load on CI and cause flakiness

Reply:
Done.